rules.ini

```
@@ -62,6 +62,11 @@ CryEngine[] = (?:^|/)cry3dengine\.dll$
 CryEngine[] = (?:^|/)CryD3DCompilerStub\.dll$
 CryEngine[] = (?:^|/)CryRenderD3D1[12]\.dll$
 CryEngine[] = (?:^|/)CryRenderVulkan\.dll$
+CyaTech[] = \.assetInfos$
+CyaTech[] = /FTT_CyaT1X
+CyaTech[] = ^Data/CyaLive_HD.\xml$
+CyaTech[] = (?:^|/)Cyatech
+CyaTech[] = /cyanide\.avi$
```

```suggestion
CyaTech[] = /FTT_CyaT1X
CyaTech[] = ^Data/CyaLive_HD.\xml$
CyaTech[] = (?:^|/)Cyatech
CyaTech[] = /cyanide\.avi$
```
should fix false positive
The Council doesn't get detected but i think internet is wrong and this isn't using cyatech engine since there isn't any mention of cyatech in any of the game files